gh-108901: Add `bound_arg` to `Signature.from_callable()` and `signature()` (#116559)
python/cpython: Doc/whatsnew/3.13.rst

built on debug mode <debug-build>`.
(Contributed by Victor Stinner in :gh:`62948`.)

inspect
-------

* Add *skip_bound_arg* parameter to :func:`inspect.Signature.from_callable`

>   and :func:`inspect.signature`, pass ``False``
>   to keep the ``self`` parameter in a signature.

Reviewer: ```suggestion
  and :func:`inspect.signature`: keep the ``self`` parameter
  in the signature if *skip_bound_arg* is False.
```

  built on debug mode <debug-build>`.
  (Contributed by Victor Stinner in :gh:`62948`.)

inspect
-------

* Add *bound_arg* parameter to :func:`inspect.Signature.from_callable`
  and :func:`inspect.signature`: keep the ``self`` parameter

>   in the method signature if *bound_arg* is True.

Reviewer 2: Reword here to match the updated main function docs (including mentioning the getfullargspec compatibility)

>   :pypi:`inspect313` package has a backport of this feature.

Reviewer 2: Maybe make the backport mention a seealso note in the module docs rather than only putting it here?
